sdk-artifact-sync.py

```diff
@@ -74,6 +76,15 @@
 if verbose:
     print 'Actual SDK m2repository folders:\n  %s' % '\n  '.join(sdk_m2_repos)
 
+auth_base64 = ''
+mvn_settings = xml.etree.ElementTree.parse(os.environ['HOME'] + '/.m2/settings.xml').getroot()
```

Comment:
This won't work cross-platform. You should use `os.path.expanduser('~')` to get the home directory and then `os.path.join` to append `.m2` and `settings.xml` to it.